packages/chart/src/CdcChart.tsx

```diff
@@ -63,7 +63,7 @@ export default function CdcChart({ configUrl, config: configObj, isEditor = fals
   const [stateData, setStateData] = useState(config.data || [])
   const [excludedData, setExcludedData] = useState<Record<string, number>[] | undefined>(undefined)
   const [filteredData, setFilteredData] = useState<Record<string, any>[] | undefined>(undefined)
-  const [seriesHighlight, setSeriesHighlight] = useState<string[]>(configObj?.legend.seriesHighlight.length ? [...configObj?.legend.seriesHighlight] : [])
+  const [seriesHighlight, setSeriesHighlight] = useState<string[]>(configObj?.legend?.seriesHighlight.length ? [...configObj?.legend?.seriesHighlight] : [])
```

Comment:
@petercolella added short circuit operator here because it's possible that configObj is null while building out a visualization for the first time. Other than that this all appears to be working well.

Reply:
@adamdoe thanks!